Lib/test/test_types.py

```diff
@@ -1456,6 +1456,7 @@ def foo():
         self.assertIs(foo(), coro)
         self.assertIs(foo().__await__(), coro)
 
+    @unittest.skip("TODO: RUSTPYTHON, unittest.mock")
```

Comment:
How did this work before?

Reply:
I'm not sure, I assume something to do with `mock` being able to easily mock methods, but us not supporting the machinery it uses to mock slots (or something like that).